EpubExtractor/Classes/EpubParser.swift

```diff
@@ -62,9 +69,8 @@ final class EpubParser {
         if let rootFileURLString = xmlDoc.root["rootfiles"].children.first?.attributes["full-path"] {
             return URL(fileURLWithPath: epubDirectoryURL.appendingPathComponent(rootFileURLString).path)
         }
-        else {
-            return nil
-        }
+        
```

Comment:
I would prefer to keep the else here, because makes it more safe
For example if I add another else and don't add a return by mistake, it will still compile
```swift
if let rootFileURLString = xmlDoc.root["rootfiles"].children.first?.attributes["full-path"] {
  return URL(fileURLWithPath: epubDirectoryURL.appendingPathComponent(rootFileURLString).path)	            
} else if X {
  _ = doX()
}

return nil
```
Compiles, but it wouldn't with the else

```swift
if let rootFileURLString = xmlDoc.root["rootfiles"].children.first?.attributes["full-path"] {
  return URL(fileURLWithPath: epubDirectoryURL.appendingPathComponent(rootFileURLString).path)	            
} else if X {
  _ = doX()
} else {
  return nil
}

Reply:
fixed in commit below:)

Reply:
I did it just in one reason - less code. 
When you do something with if - else conditiion, and you have only one IF, then return, and one else that return something default - I remove "else" condition because there it is unnecessary 

If there are more than one condition - it's true, always use else condition